src/release_artifacts_resources/ios/cdk/cdk/credential_rotation/utils/secretmanager_constants.py

```diff
@@ -0,0 +1 @@
+CIRCLECI_API_KEY = "CIRCLECI_API_KEY"
```

Comment:
Add a comment enumerating the places these are hardcoded (e.g., the Lambda rotator function)

Reply:
Added